### PR TITLE
Use JS instead of TS assets.

### PR DESCRIPTION
**Pull request by dkrichards86**
## Jira
[AUTH-779](https://clever.atlassian.net/browse/AUTH-779)

## Overview
This branch uses `tsc` to precompile TypeScript assets at Docker build time.

## Testing
Smoke testing in dev

## Rollout
:100:

[AUTH-779]: https://clever.atlassian.net/browse/AUTH-779?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by dkrichards86** [comment]
This is currently deployed to clever-dev.

**Review by ulziibay — COMMENTED** [review]
why did you have to move `tracing.ts` to `tracing.js`?

**Comment by ulziibay** [comment]
https://clever-dev--who-is-who--1775dedb.int.clever.com/all seems fine!

**Review by ulziibay — APPROVED** [review]
one question, otherwise looks good!

File: package.json [hunk 1]
```diff
@@ -25,8 +24,6 @@
     "express": "^4.17.3",
     "kayvee": "^3.14.0",
     "lodash": "^4.17.21",
-    "ts-node": "^9.1.1",
-    "typescript": "^4.2.4",
```
Comment (ulziibay) [inline package.json:29]: why are we removing this?

**Comment by dkrichards86** [comment]
@ulziibay I moved `tracing.ts` to `tracing.js` because it was the only TS file in the entire service, and it was instantiating opentelemetry. We didn't gain anything from types there.

File: package.json (hunk 1 above)
Comment (dkrichards86) [reply package.json:29]: The only TS file was tracing.ts, and it just instantiated otel. We didn't gain anything from having types in that file. I moved it to JS so now all files are vanilla JS. No need to precompile if we don't have any TS.

**Comment by ulziibay** [comment]
@dkrichards86 
> No need to precompile if we don't have any TS.

Alt Text

**Pull request merged**